deploy.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p jq -i bash
+
+# shellcheck shell=bash
+
+set -euo pipefail
+
+SSHOPTS=()
+export NIX_SSHOPTS="${SSHOPTS[*]}"
+
+# For first deployment to bare server
+if [[ ${1:-} == '--prime' ]]; then
+    shift
+
+    BARE_SERVER=1
+else
+    BARE_SERVER=0
+fi
+
+get() {
+    jq -r ".$1" <<< "$MERGED"
+}
+
+check_set() {
```

Comment:
A more fitting name would probably be `ensure_set` or `assert_set`, given that it aborts execution.